opencog/python/preload_functions/test_functions.py

```diff
@@ -1,6 +1,11 @@
 # 
 # Test Functions preloaded into the CogServer
 
+# Modify PYTHON_PATH so that test_helpers module gets registered.
+from sys import path 
+from os.path import realpath
+path.append(realpath('..')+"/opencog/python")
```

Comment:
well, the `cogserver.conf` file should already be doing this.  That is, the correct path should already be searched, so this hack should not be needed.  Maybe cogserver.conf is broken?   But whatever. Path stuff in opencog is a horrible mess.